api/metadata_handler.go

```diff
@@ -147,3 +235,12 @@ var cacheKey = func(r *http.Request) interface{} {
 
 	return fmt.Sprintf("%s#%s", r.URL.String(), string(data))
 }
+
+func getWPTFYIGithubBot(ctx context.Context) (*github.Client, error) {
```

Comment:
I don't see much use in this function. Without it, the code on line 61 just looks like:

```
	githubBotClient, err := shared.GetGithubClientFromToken(ctx, "github-wpt-fyi-bot-token")
	if err != nil {
		http.Error(w, "Unable to get Github Client: "+err.Error(), http.StatusInternalServerError)
		return
	}
```

Which seems very reasonable to me, and saves us adding this function :). If we had more users of the function, might be a different story (to avoid typing 'github-wpt-fyi-bot-token' over and over)